packages/fxa-graphql-api/src/test/lib/oauth/index.spec.ts

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Container } from 'typedi';
+import { Request, Response } from 'express';
+import { assert } from 'chai';
+import 'mocha';
+import sinon from 'sinon';
+
+import { loggerContainerKey, configContainerKey } from '../../../lib/constants';
+
+const sandbox = sinon.createSandbox();
+// tslint:disable-next-line: no-empty
+const mockLogger = { debug: () => {} };
```

Comment:
Would it be helpful to make this a sinon spy/stub so we can assert things on it?